src/stories/Colors/colors.mdx

```diff
@@ -9,27 +9,19 @@ import Styles from "./colors.module.css";
 ## Primary Colors
 
```

Comment:
did you also run storybook ? if so, nice to add screenshots in description as well! 🎉

Reply:
Added now :+1: